### DISCUSSION TITLE
Best way to implement a badge in a placeholder?

### DISCUSSION THREAD

**Discussion by Jam0r85** [filamentphp/filament#6563] [Help] [answered]
Any suggestions as to the best way to implement a badge for a form placeholder?

Something that would match the badge used in the table component? Or could that even be used?

Mainly for edit/view pages where you need something to stand out a bit more rather than just plain text. I know you can return a HtmlString object in the content of a placeholder but having to do that again and again will be a pain to maintain in the future.

**Comment by danharrin** [comment, marked as answer]
It would be a custom field 👍 Full control over the view

**Reply by danharrin** [reply]
In the view, `{{ $getState() }}` is the value in Blade
